game/scripts/npc/items/item_ancient_janggo_of_endurance_2.txt

```diff
@@ -53,11 +53,12 @@
     "ItemShopTags"                                        "str;agi;int;damage;move_speed;attack_speed;hard_to_tag"
     "ItemQuality"                                         "rare"
     "ItemAliases"                                         "drum of endurance"
-    "ItemInitialCharges"                                  "6"
-    "ItemDisplayCharges"                                  "1"
+    //  Infinite Charges:
+    "ItemInitialCharges"                                  "-1"
```

Comment:
thats how it was in the level 5, i just made it consistent, since level 5 works as intended i just had to copy.